host_modules/gnoi_reboot.py

```diff
@@ -12,13 +12,12 @@
 # Both enum and string representations are supported
 REBOOTMETHOD_COLD_BOOT_VALUES = {1, "COLD"}
 REBOOTMETHOD_WARM_BOOT_VALUES = {4, "WARM"}
-REBOOTMETHOD_NSF_VALUES = {5, "NSF"}
 
 # Timeout for SONiC Host Service to be killed during reboot
 REBOOT_TIMEOUT = 260
 
 EXECUTE_COLD_REBOOT_COMMAND = "sudo reboot"
-EXECUTE_NSF_REBOOT_COMMAND = "/etc/init.d/gpins-nsf-boot nsf-reboot"
+EXECUTE_WARM_REBOOT_COMMAND = "/usr/local/bin/warm-reboot -v"
```

Comment:
@hdwhdw @qiluo-msft Can you pls confirm this is the right command to invoke warm reboot in SONiC ?

Reply:
Yess. https://github.com/sonic-net/sonic-utilities/blob/34d2c8ce007b077b75a29ade8f0876374f66f21f/scripts/warm-reboot#L1